buckbuild.bzl

```diff
@@ -939,6 +939,7 @@ def define_buck_targets(
                 ("", "torch/csrc/**/*.h"),
                 ("", "torch/script.h"),
                 ("", "torch/library.h"),
+                ("", "torch/library_stable.h"),
```

Comment:
I would prefer a new `stable` subfolder to make this easier to track

Reply:
ya this was just i could get some progress in, will figure out the build stuff soon